special-interest-groups/sig-planner/member-list.md

```diff
@@ -0,0 +1,22 @@
+# SIG Planner Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2019-11-24
```

Comment:
Do we need to maintain this field?
The update history is maintained by github too.